packages/webview_flutter/webview_flutter_android/example/lib/main.dart

```diff
@@ -127,6 +127,11 @@ Page resource error:
           })
           ..setOnUrlChange((UrlChange change) {
             debugPrint('url change to ${change.url}');
+          })
+          ..setOnHttpAuthRequest((HttpAuthRequest request) {
+            debugPrint(
```

Comment:
Does this add any value given that the dialog will include the same information?